### ISSUE TITLE
Email template accessibility issues

### ISSUE THREAD

**Issue by CarlSchwan** [nextcloud/server#33335]
- [ ] Button has no focus effect
- [ ] Footer contrast is very bad
- [ ] ...

**Comment by skjnldsv** [comment]
Superseded by https://github.com/nextcloud/server/issues/48095 
New template in development
